Documentation/configuration.md

```diff
@@ -1,8 +1,8 @@
 # rkt configuration
 
-`rkt` reads configuration from two directories - a **system directory** and a **local directory**.
-The system directory defaults to `/usr/lib/rkt`, and the local directory to `/etc/rkt`.
-Both locations can be changed with command line flags.
+`rkt` reads configuration from two or three directories - a **system directory**, a **local directory** and, if provided, a **user directory**.
+The system directory defaults to `/usr/lib/rkt`, the local directory to `/etc/rkt`, and the user directory to empty string.
+These locations can be changed with command line flags.
```

Comment:
It should explain what is system, local and user in this context, and why it is useful to have several directories.

- system: config from the distribution, read-only. The admin cannot change it, or is not supposed to.
- local: (local to the machine) can be modified by an admin.
- user: more specific to a user. Uses: when there are several users fetching images with different http credentials. Or when specifying a different directory for the network plugin that should not be used by all rkt instances on this machine.